spring-ai-spring-boot-autoconfigure/src/main/java/org/springframework/ai/autoconfigure/chat/observation/ChatObservationAutoConfiguration.java

```diff
@@ -113,6 +124,17 @@ ChatModelCompletionObservationFilter chatModelCompletionObservationFilter() {
 
 	}
 
+	@Bean
+	@ConditionalOnBean(Tracer.class)
+	@ConditionalOnProperty(prefix = ChatObservationProperties.CONFIG_PREFIX, name = "include-error-logging",
+			havingValue = "true")
+	public ErrorLoggingObservationHandler errorLoggingObservationHandler(Tracer tracer) {
+		return new ErrorLoggingObservationHandler(tracer,
+				List.of(EmbeddingModelObservationContext.class, ImageModelObservationContext.class,
```

Comment:
is it ok for the property `pring.ai.chat.observations.include-error-logging` to apply to all these context's as compared to the chatmodel one, e.g. embedding?